### DISCUSSION TITLE
excessive redirection using cloudflare tunnel

### DISCUSSION THREAD

**Discussion by NellyOfc** [mastodon/mastodon#24804] [Q&A]
hello, when I try to use cloudflare tunnel to make a tunnel from mastodon to a domain using tunnel, it gives an excess redirection problem, does anyone know how I can solve it?

**Comment by NellyOfc** [comment]
I managed to solve it, but it keeps giving a lot of errors, I can't post messages or send files and images

![image](https://user-images.githubusercontent.com/87354471/235839800-81a19332-4150-4e45-a667-75aa95c44d3a.png)

**Reply by 0wg** [reply]
How did you solve it?

**Comment by NellyOfc** [comment]
error 500 media

![image](https://user-images.githubusercontent.com/87354471/235881710-fd9a048a-bee5-439e-9fa5-56064f1f7244.png)

**Comment by Longtianmu** [comment]
Well I think there are some SSL error actually.
Do not use Flex SSL mode in cloudflare.

**Reply by 0wg** [reply]
Tried that, did not work
